### PR TITLE
SC-591 Overloaded Issue action constructor

### PR DESCRIPTION
**Pull request by xrtm000**

### CODE REVIEW

File: lang/doc/v4/funcs/internal-functions.hjson [hunk 1]
```diff
@@ -5,6 +5,18 @@
...
       paramsDoc: [ "The value", "The type name" ]
+    },
+    {
+      name: "Issue"
+      params: [ "String", "String", "Int", "Int", "Boolean", "Script|Unit", "Int" ]
+      doc: "Issue action detailed constructor"
+      paramsDoc: [ "Token name", "Token description", "Amount of the token", "Number of digits in decimal part", "Reissue ability flag", "Smart asset script (optional)", "Sequential number"]
+    },
+    {
+      name: "Issue"
```
Comment (msmolyakov) [inline lang/doc/v4/funcs/internal-functions.hjson:16]: are they will be shown in the surfboard help, for example?

File: lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Functions.scala [hunk 2]
```diff
@@ -534,4 +534,34 @@ object Functions {
...
+      FieldNames.IssueQuantity -> LONG,
+      FieldNames.IssueDecimals -> LONG,
+      FieldNames.IssueIsReissuable -> BOOLEAN,
+    ) {
+      args =>
+        val typedArgs = (issueActionType.fields.map(_._1) zip (args ::: List(unit, CONST_LONG(0)))).toMap
+        Right(CaseObj(issueActionType, typedArgs))
+    }
+
+  val detailedIssueActionConstructor: BaseFunction[Environment] =
```
Comment (msmolyakov) [inline lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Functions.scala:552]: Is the origin constructor of the Issue structure removed from the context?

Comment (xrtm000) [reply lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Functions.scala:552]: Not removed but overlapped

File: lang/doc/v4/funcs/internal-functions.hjson (hunk 1 above)
Comment (xrtm000) [reply lang/doc/v4/funcs/internal-functions.hjson:16]: should be

**Pull request merged**